# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/indexedDB/FileManager.h
# Commit: 8a21ea441fc7
# Full Hash: 8a21ea441fc78be36c13f78fc3da9fb41985ed3a
# Author: Simon Giesecke <sgiesecke@mozilla.com>
# Date: 2020-05-14 16:15:31
# Regressor Bug: 1623278
# File Overlap Count: 1
# Description:
#   Bug 1623278 - Use MovingNotNull in ActorsParent. r=dom-workers-and-storage-reviewers,janv
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D73033
# ==============================================================================

diff -r ab6a041c0d57 -r 8a21ea441fc7 dom/indexedDB/FileManager.h
--- a/dom/indexedDB/FileManager.h	Thu May 14 07:47:46 2020 +0000
+++ b/dom/indexedDB/FileManager.h	Wed May 13 10:44:52 2020 +0000
@@ -68,7 +68,7 @@
 
   bool EnforcingQuota() const { return mEnforcingQuota; }
 
-  nsresult Init(nsIFile* aDirectory, mozIStorageConnection* aConnection);
+  nsresult Init(nsIFile* aDirectory, mozIStorageConnection& aConnection);
 
   [[nodiscard]] nsCOMPtr<nsIFile> GetDirectory();
 
